kong/plugins/prometheus/exporter.lua

```diff
@@ -71,6 +75,23 @@ local function init()
                                          "Total bandwidth in bytes " ..
                                          "consumed per service/route in Kong",
                                          {"service", "route", "type"})
+
+
+  -- Hybrid mode status
+  if kong.version_num >= 2000000 then -- 200.00.00 -> 2.0.0 then
```

Comment:
Is this check needed? `kong.configuration.role` will be nil in previous versions and that should be sufficient.

Reply:
will remove this check